packages/formatter-html/src/result.ts

```diff
@@ -4,10 +4,11 @@ import * as moment from 'moment';
 
 import cloneDeep = require('lodash/cloneDeep');
 
+import { fs } from '@hint/utils';
```

Comment:
Nit: Can't we just do `import { loadJSONFile } from '@hint/utils/fs';` or something similar?